src/clj/tasks/nrdb.clj

```diff
@@ -95,7 +92,7 @@
 
 (defn fetch-data
   [{:keys [card-images db local]}]
-  (let [edn (download-edn-data local)]
+  (let [edn (dissoc (download-edn-data local) :promos)]
```

Comment:
We can just remove the `:promos` data from netrunner-data.

Reply:
Agree. Until it is removed from `netrunner-data` I need to filter it. After it's gone we should take this out.